Cropping faces from image/Cropping faces from image.py

```diff
@@ -0,0 +1,36 @@
+import cv2                           #importing cv library
+import numpy as np                   #importing numpy library as np   
+import os                            #importing os module
+
+path = input('Enter the path for image: ')   #Taking image path from user
+
+img = cv2.imread(path)
+  
+# Convert image into grayscale
+gray = cv2.cvtColor(img, cv2.COLOR_BGR2GRAY)   
+
+#cascade loading
+face_cascade = cv2.CascadeClassifier('haarcascade_frontalface_alt2.xml')
+
+face_cascade = cv2.CascadeClassifier(cv2.data.haarcascades + 'haarcascade_frontalface_default.xml')
```

Comment:
Why this is loaded two times? Also add the file `haarcascade_frontalface_default.xml` used in this script